### PR TITLE
Add OS charges on a paid plan

### PR DESCRIPTION
**Pull request by ivanaur**

### CODE REVIEW

File: docs/account-management/plans.md [hunk 1]
```diff
@@ -33,7 +33,12 @@ out to
 [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com)
 (please include which type of machine you prefer).
 
-**Note:** In Semaphore 2.0, all organizations are on monthly plans.
+In Semaphore 2.0, all organizations are on monthly plans.
+
+**Note:** If you are building open source projects on a Paid plan, those builds 
+will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
```
Comment (Ivana-Markovic) [inline docs/account-management/plans.md]: "If you are building open source projects on a Paid plan, the charges defined on our [pricing](https://semaphoreci.com/pricing) page will apply." **or** "If you are building open source projects on a Paid plan, you will be charged according to our [pricing](https://semaphoreci.com/pricing) page."

pls note that you have a typo in 'according' (accourding)

File: docs/account-management/plans.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ out to
 
 In Semaphore 2.0, all organizations are on monthly plans.
 
-**Note:** If you are building open source projects on a Paid plan, those builds 
-will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
-Open source projects can be built free of charge only on an [Open Source
+**Note:** If you are building open source projects on a Paid plan, you will be 
+charged accourding to our [pricing page](https://semaphoreci.com/pricing) for those
```
Comment (Ivana-Markovic) [inline docs/account-management/plans.md]: accourding => according

File: docs/account-management/plans.md [hunk 3]
```diff
@@ -49,7 +49,8 @@ machine at your disposal.
 Only one of the organizations you own can be on an open source plan.
 
 **Note:** Open source projects can be built free of charge only on an Open Source
-plan. If you are building open source projects on a [Paid plan](https://docs.semaphoreci.com/account-management/plans/#paid-plan), those builds will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
+plan. If you are building open source projects on a [Paid plan](https://docs.semaphoreci.com/account-management/plans/#paid-plan), you will be charged accoring to our [pricing page](https://semaphoreci.com/pricing)
```
Comment (Ivana-Markovic) [inline docs/account-management/plans.md]: accoring => according

File: docs/account-management/plans.md [hunk 4]
```diff
@@ -49,7 +49,8 @@ machine at your disposal.
...
+for those builds.
```
Comment (Ivana-Markovic) [inline docs/account-management/plans.md]: those is more like 'oni bildovi' and we need 'za te bildove' so I would change 'those builds' to 'these builds'.

File: docs/account-management/plans.md [hunk 5]
```diff
@@ -35,9 +35,9 @@ out to
 
 In Semaphore 2.0, all organizations are on monthly plans.
 
-**Note:** If you are building open source projects on a Paid plan, those builds 
-will be charged accourding to our [pricing](https://semaphoreci.com/pricing).
-Open source projects can be built free of charge only on an [Open Source
+**Note:** If you are building open source projects on a Paid plan, you will be 
+charged accourding to our [pricing page](https://semaphoreci.com/pricing) for those
+builds, too. Open source projects can be built free of charge only on an [Open Source
```
Comment (Ivana-Markovic) [inline docs/account-management/plans.md]: those is more like 'oni bildovi' and we need 'za te bildove' so I would change 'those builds' to 'these builds'.

File: docs/account-management/plans.md (hunk 2 above)
Comment (ivanaur) [reply docs/account-management/plans.md]: Sorry I missed it again 😬

**Pull request merged**